Scripts/RemediationScripts/Remediate-ConfigureConditionalAccessPolicyForPIM.ps1

```diff
@@ -204,7 +208,6 @@ function Configure-ConditionalAccessPolicyForPIM
 
         [String]
         [Parameter(ParameterSetName = "WetRun", Mandatory = $false, HelpMessage="Specifies the role name on which CA policy needs to be configured")]
-        [ValidateSet("Owner", "Contributor", "User Access Administrator")]
         $RoleName,
```

Comment:
so we are accepting any role name now? have we put any check to see if it's valid role name or not?

Reply:
No we are not, I have just removed the validation set from here, but still validation is there, attaching screenshot for reference.
![image](https://github.com/azsk/AzTS-docs/assets/141649476/c10f9670-8cc3-4e94-b285-633c743173ca)